B/Birch/build_tarballs.jl

```diff
@@ -0,0 +1,96 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Birch"
+version = v"1.0.0"
```

Comment:
Actually, this PR builds the current master of Birch and not the tag "v1" since according to Birch's README "The master branch is considered stable".